src/components/legacy/typography.tsx

```diff
@@ -27,7 +27,7 @@ export const SmallTitle = styled.h5`
  * Text
  *
  */
-export const Text = styled.p`
+export const Text = styled.div`
```

Comment:
I can confirm this fixes the console error 🙌 However, normal paragraphs in the blog now also have the `<div>` tag, which is not ideal. Is there a way to ensure that they are still `<p>`? 

<img width="466" alt="Screenshot 2024-11-14 at 14 30 00" src="https://github.com/user-attachments/assets/9f3e7c67-937e-455d-beca-b90e9dc84723">

Reply:
contentful doesnt differentiate text and just paragraphs wrapped around the youtube embed. I tried solving it by just changing `<p>`s with a `<div>` in them to a `<div>` too but the error was triggered before that code finished executing. because in this case the only difference between a `<div>` and a `<p>` is the semantics i think i should be to just use `<div>`s